src/scss/components/_register-comp.scss

```diff
@@ -0,0 +1,204 @@
+.form {
+  background: rgb(255, 206, 70);
+  padding-top: 115px;
+
+  @include viewport-desktop {
+    padding-top: 260px;
+  }
+
+  &__container {
+    @extend .grid;
```

Comment:
Please do not extend the class if you are not adding new properties. If you need to use the `.grid` class, add it to the element that needs it.

Reply:
Done!